src/entities/activity/model/mappers.ts

```diff
@@ -147,7 +147,7 @@ function mapToAudio(dto: AudioItemDto): ActivityItem {
     config: {
       maxDuration: dto.responseValues.maxDuration,
     },
-    timer: mapTimerValue(dto.timer),
+    timer: mapTimerValue(dto.config.timer),
```

Comment:
Backend has changed times placement in dto-s? was it mentioned anywhere? or added into documentation?